src/view/container/device/Device.tsx

```diff
@@ -53,17 +54,71 @@ const CPX_TOOLBAR_BUTTONS: Array<{ label: any; image: any }> = [
         label: TOOLBAR_ICON_ID.GPIO,
     },
 ];
-
+interface IState {
+    currentDevice: string | undefined;
+}
+interface IProps {
+    children?: any;
+}
+const DEVICE_LIST_KEY = {
+    CPX: "cpx",
+    MICROBIT: "microbit",
+};
 // Container to switch between multiple devices
 
-class Device extends React.Component {
+class Device extends React.Component<IProps, IState> {
+    constructor(props: IProps) {
+        super(props);
+        this.state = {
+            currentDevice: DEVICE_LIST_KEY.MICROBIT,
```

Comment:
Did you confirm with PM about this, to have Microbit as the default one?

Reply:
@sayyedamussa

Reply:
@sagarmanchanda yes we want to make It default

Reply:
We shouldn't make micro:bit the default one until it's full featured.

Reply:
And once micro:bit is full featured, we should not force the change on existing users

Reply:
Got it.

Reply:
Sounds good - @sagarmanchanda @xnkevinnguyen  lets not make microbit to default